core/class/scenarioExpression.class.php

```diff
@@ -1759,6 +1759,12 @@ public function execute(&$scenario = null) {
 					}
 					return;
 				} elseif ($this->getExpression() == 'remove_inat') {
+					if (isset($options['scenario_id']) && intval($options['scenario_id']) != 0) {
+						$targetScenario = scenario::byId($options['scenario_id']);
```

Comment:
It would be good to add error handling for the case when the target scenario is not found. This would prevent potential issues down the line. [important]

Reply:
No, if target scenario is not found, `remove_inat` previous behavior is followed: `execute()` returns.
But I can change that if you want?